pkg/redshift/api/api.go

```diff
@@ -89,6 +91,7 @@ func (c *API) Execute(ctx context.Context, input *api.ExecuteQueryInput) (*api.E
 		DbUser:            commonInput.DbUser,
 		SecretArn:         commonInput.SecretARN,
 		Sql:               aws.String(input.Query),
+		WithEvent:         commonInput.WithEvent,
```

Comment:
minor thing. Since the `withEvent` is only available for the `Execute` method, I would not include it in the `commonInput` structure.